nvm.sh

```diff
@@ -660,6 +661,39 @@ nvm_checksum() {
   fi
 }
 
+# For iojs, with a view to supporting SHA256 for Node > 0.10 soon as well.
+# https://github.com/creationix/nvm/pull/664
+nvm_checksum_256() {
+  local NVM_CHECKSUM
+
+  if nvm_has "sha256sum" && ! nvm_is_alias "sha256sum"; then
```

Comment:
Would some debug output be helpful here to indicate which of these many branches is being run on travis? Specifically, which sha command is being used?